CHANGELOG.md

```diff
@@ -1,16 +1,19 @@
 # Lotus changelog
 
-# v1.13.2-rc3 /  2021-12-09
+# v1.13.2-rc4 /  2021-12-10
 
-This is the second RC for lotus v1.13.2, with a dependency upgrade of ipld-legacy which will fix ChainGetNode. This 
-is a highly recommended release with sealing pipeline fixes, worker management and scheduler enhancement, retrieval improvements and so on. More detailed changelog will be updated later.
+This is the 4th RC for lotus v1.13.2, with another retrieval enhancement that fills the gap that;s brought by the release. This is a highly recommended release with sealing pipeline fixes, worker management and scheduler enhancement, retrieval improvements and so on. More detailed changelog will be updated later.
 
 - github.com/filecoin-project/lotus:
-  - CARv2 v2.1.0
-  - dagstore pieceReader: Fix wrong ErrUnexpectedEOF return in ReadAt
-  - dagstore pieceReader: Always read full in ReadAt
-  - Add metrics to dagstore piecereader
-  - disable building of appimage on release
+  - stores: Reduce log spam during retrievals
+  - fix lint
```

Comment:
would remove fix lint from changelog

Reply:
this is a cherry-picked commit   - in the final changelog it will be updated with the pr!